internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -777,9 +777,15 @@ func (c *ClusterManager) buildResourceSet(cluster api.Cluster) types.ResourceSet
 		c.buildObservabilityCatalogSourceResource(),
 		c.buildObservabilityOperatorGroupResource(),
 		c.buildObservabilitySubscriptionResource(),
-		c.buildObservabilityRemoteWriteServiceAccountCredential(&cluster),
```

Comment:
Are `buildObservatoriumDexSecretResource` and `buildObservatoriumSSOSecretResource` still needed with the proxy?

Reply:
The dex secret will be removed in a separate PR. The RHSSO secret is still created, but fields containing credentials are set to empty strings. Can be further improved in a separate PR.